### DISCUSSION TITLE
Website txt file - 3rd party host we can't access root folder - help?

### DISCUSSION THREAD

**Discussion by BsadlerSRMC** [CMSgov/hospital-price-transparency#15] [Q&A] [answered]
Hello,
Our PR team is stating a 3rd party hosts the design of our website and the PR staff cannot directly access the root folder. Is there an alternative solution if we cannot access the root folder to adhere to the requirements?
Thank you,
Brittney - Salem Community Hospital

**Comment by shaselton-usds** [comment]
As finalized in CY2024 OPPS/ASC Final Rule, beginning January 1, 2024, each hospital must ensure that the public website it selects to host its machine-readable file (MRF) establishes and maintains, in the form and manner specified by CMS:

- A .txt file in the root folder that includes:
  - The hospital location name that corresponds to the machine-readable file;
  - The source page URL that hosts the machine-readable file;
  - A direct link to the machine-readable file (the machine-readable file URL); and
  - Hospital point of contact information.
- A link in the footer on its website, including but not limited to the homepage, that is labeled ‘‘Price Transparency’’ and links directly to the publicly available web page that hosts the link to the machine-readable file.

Please refer to 45 CFR 180.50 (d)(6) and discussion at 88 FR 82111-82113.

**Reply by maven** [reply]
This response is not an answer to the question above, from @BsadlerSRMC . Our hospital is also unable to access the root folder. What alternatives are available besides placing a `cms-hpt.txt` file at the root level of the domain? We are able to point to Price Transparency in our footer, however it is not possible to place a requested text file at the root level.

Thank you.

**Reply by dmercer-sch** [reply]
@maven @BsadlerSRMC This is not the first post I have read where the "answer" seems to not answer anything close to what the question is. Depending on your web host, you should be able to create the txt file (if you can figure out what the info they want is) and upload it to that folder, however, you may need to contact your web host's customer service for help with that as it is not a simple task.

**Reply by RidenourK** [reply, marked as answer]
@maven @BsadlerSRMC Thank you for your questions. CMS recognizes this issue and additional guidance will be forthcoming.

**Reply by danaw999** [reply]
Can you link to the additional guidance?

**Comment by PTTCorroHealth** [comment]
We received a call today from CMS about my ticket regarding Wix/other 3rd party platforms and the .TXT file.

CMS informed me that they will be looking into options/workarounds on their end to help. They are unsure how this might go, but they understand the undertaking it would be to change and develop an entire new site, just to meet this requirement.

I would email them at HPTCompliance@cms.hhs.gov and let them know your situation so if they issue you a violation letter you can respond and remind them of your issue.

**Comment by RidenourK** [comment]
@BsadlerSRMC @maven @PTTCorroHealth 

Thank you for your patience. We recommend reviewing the updated [Hospital Price Transparency TXT File FAQs](https://www.cms.gov/files/document/hospital-price-transparency-txt-file-frequently-asked-questions-faqs.pdf) which contain new guidance for addressing the specific challenges you’re facing.

**Reply by nmaylevc** [reply]
The new TXT File FAQs still doesn't address the issue.

In most cases that we have seen the issue is that the site used to host the MRFs doesn't allow access to putting a file in the "root folder".  This is much less often the case with the hospital's own site.  A simple modification that would address the issue for most hospitals would be to allow the file to be placed at the root of the website hosting the MRFs, but if that isn't possible than in the root of the hospital's website.

Generally, the more sensible place for this TXT file is on the hospital's own website.  The stated goal of this "new" rule was to make the MRFs more easily findable and putting the file that allows you to find the MRF on some hard-to-predict cloud storage site isn't the best solution.  The problem now is: When I am looking for the cms-hpt.txt file for a hospital, where on the web should I look?  I am back to using a search engine or trying to find it using urls I find on the page pointed to by the footer link.

The more logical place for the file that tells you where to find the MRFs is in a predictable place: each hospital's own website.  Then we would all know where to look for this file for any given hospital.

**Comment by support-gchd** [comment]
Our hosting company, Weebly does not support placing the .txt file in the root directory and also does not support a redirect option. We have been put on notice and have to provide a corrective action within 45 days and resolutions withing 90 days. How can I provide a corrective action when our hosting company doesn't allow either solution provided by CMS? CMS has cancelled a scheduled meeting and didn't re-schedule. My contact is now out of office until June 24th. This is very frustrating as I do not have any options with our current Hosting company to be in compliance with this request and am being forced to provide a corrective action. Is there anyone who will help address our situation? We are the smallest of hospitals in the entirety of the US.

**Reply by maven** [reply]
I'm not with CMS, but I wanted to send over some help your way.

1. CMS did update their FAQ doc and added the redirect support: 
[https://www.cms.gov/files/document/hospital-price-transparency-txt-file-frequently-asked-questions-faqs.pdf](https://www.cms.gov/files/document/hospital-price-transparency-txt-file-frequently-asked-questions-faqs.pdf)

2. Here is the step-by-step for setting up the redirect: 
[https://www.cms.gov/files/document/wix-url-redirect.pdf](https://www.cms.gov/files/document/wix-url-redirect.pdf)

3. You will notice the Wix redirect editor is similar to Weebly. 
[https://www.weebly.com/app/help/us/en/topics/how-to-301-redirect-an-old-page-to-a-new-weebly-page](https://www.weebly.com/app/help/us/en/topics/how-to-301-redirect-an-old-page-to-a-new-weebly-page)

https://www.weebly.com/seo/redirects#:~:text=If%20you%20move%20domains%20to,as%20best%20practice%20for%20SEO.

I hope this helps...

**Reply by nmaylevc** [reply]
The item in the FAQ about redirects doesn't address the case where the MRFs are stored on some sort of web file service like google drive, one drive, dropbox etc.  The current language still says the cms-hpt.txt files should live on the site that hosts the MRFs.

**Reply by support-gchd** [reply]
Thank you. The problem is Weebly only allows "page" redirects and not "file" redirects which is what is being required. I have confirmed this with many hours of interaction with Weebly's support.

**Reply by maven** [reply]
If you want to contact me directly, I would be more than happy to assist you. [mavencatalyst.com](https://www.mavencatalyst.com)

**Reply by danaw999** [reply]
> Thank you. The problem is Weebly only allows "page" redirects and not "file" redirects which is what is being required. I have confirmed this with many hours of interaction with Weebly's support.

Was a Weebly resolution found?  I'm curious if anyone can provide instructions for Weebly instead of Wix?

**Reply by support-gchd** [reply]
Hi, Sadly Weebly did not have a solution and stated they had no plans to allow a file redirect option. We were forced to migrate to Wix due to this CMS requirement.

**Comment by PTTCorroHealth** [comment]
CMS Posted a Document - https://www.cms.gov/files/document/wix-url-redirect.pdf
Hope this helps

[CMS-wix-url-redirect-508.pdf](https://github.com/user-attachments/files/16473892/CMS-wix-url-redirect-508.pdf)

**Comment by PCHD156** [comment]
Does anyone know if there has been an update to a workaround for websites that won't allow a redirect or placement of the txt file in the root directory?

**Comment by support-gchd** [comment]
We were forced to migrate to WIX.

**Reply by PCHD156** [reply]
Thanks for your reply.  

If anyone has a different resolution, please do post it.  Some facilities cannot afford Wix.
